homeassistant/components/onvif/camera.py

```diff
@@ -258,6 +255,14 @@ def __init__(self, hass, config):
                           self._name, err)
             return
 
+    async def async_setup_ptz(self):
```

Comment:
This method is marked as a coroutine but it doesn't await any tasks - are the `.get_service` etc calls blocking calls? If so, they should be awaited and not run in the event loop.

Reply:
Maybe, I just moved this code to a method. 
what shoud we do?

Reply:
That need not be a coroutine because you don't use await inside.